duckdb/README.md

```diff
@@ -1,6 +1,26 @@
 # DuckDB Data Connector
 
```

Comment:
Please add 

Clone the Spice quickstarts repository and navigate to the `duckdb` directory:

```bash
git clone https://github.com/spiceai/quickstarts.git
cd quickstarts/duckdb
```

Reply:
Used spice init instructions instead of clone. User may not clone the whole repository for Quickstart but init will work